nest-tax-backend/src/utils/guards/errors.guard.ts

```diff
@@ -1,132 +1,180 @@
-        messageSk: 'Daň s týmto ID nebola nájdená, prosím kontaktujte support.',
-        errorName: CustomErrorPaymentTypesEnum.TAX_NOT_FOUND,
-      },
-      422,
+  UnprocessableEntityException(
+    errorEnum: CustomErrorEnums,
+    message: string,
+    status?: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    return this.LoggingHttpException(
+      HttpStatus.UNPROCESSABLE_ENTITY,
+      errorEnum,
+      message,
+      status || 'Unprocessable entity',
+      console,
+      object,
     )
   }
 
-  qrCodeNotFound() {
-    return new HttpException(
-      {
-        statusCode: 422,
-        status: 'Not Found',
-        message: 'QR code was not found',
-        messageSk:
-          'QR kód nebol pre túto daň nájdený, prosím kontaktujte support.',
-        errorName: CustomErrorPaymentTypesEnum.QR_CODE_NOT_FOUND,
-      },
-      422,
+  NotFoundException(
+    errorEnum: CustomErrorEnums,
+    message: string,
+    status?: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    return this.LoggingHttpException(
+      HttpStatus.NOT_FOUND,
+      errorEnum,
+      message,
+      status || 'Not found',
+      console,
+      object,
     )
   }
 
-  paymentResponseRedirectError(error: any): HttpException {
-    this.logger.error(error)
-    return new HttpException(
-      {
-        statusCode: 422,
-        status: 'Error to redirect',
-        message: 'Error to redirect to response',
-        errorName: CustomErrorPaymentResponseTypesEnum.PAYMENT_RESPONSE_ERROR,
-      },
-      422,
+  BadRequestException(
+    errorEnum: CustomErrorEnums,
+    message: string,
+    status?: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    return this.LoggingHttpException(
+      HttpStatus.BAD_REQUEST,
+      errorEnum,
+      message,
+      status || 'Bad Request',
+      console,
+      object,
     )
   }
 
-  renderPdfError(error: any): HttpException {
-    this.logger.error(error)
-    return new HttpException(
-      {
-        statusCode: 422,
-        status: 'Error to create pdf',
-        message: 'Error to create pdf',
-        errorName: CustomErrorPdfCreateTypesEnum.PDF_CREATE_ERROR,
-      },
-      422,
+  UnauthorizedException(
+    errorEnum: CustomErrorEnums,
+    message: string,
+    status?: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    return this.LoggingHttpException(
+      HttpStatus.UNAUTHORIZED,
+      errorEnum,
+      message,
+      status || 'Unauthorized',
+      console,
+      object,
     )
   }
+
+  private LoggingHttpException(
+    statusCode: number,
+    errorEnum: CustomErrorEnums,
+    message: string,
+    status: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    const response: ResponseErrorInternalDto = {
+      statusCode,
+      status,
+      errorName: errorEnum,
+      [Symbol('alert')]: alertReporting.includes(errorEnum) ? 1 : 0,
```

Comment:
I think the intended usage for symbols in JS is mainly that you create symbol in one place:
```
export const alertSymbol = Symbol('alert')
```
and then you work with the exported symbol to retrieve it.